backend/src/main/java/edu/nus/market/service/AccountServiceImpl.java

```diff
@@ -29,9 +30,37 @@ public class AccountServiceImpl implements AccountService{
     @Resource
     PasswordHasher passwordHasher;
 
+    /**
+     *
+     * @param id
+     * @return ResponseEntity
+     * @author jyf
+     */
+    @Override
+    public ResponseEntity<Object> getMyAccount(int id) {
+        Account account = accountDao.getAccountById(id);
+        if (account == null)
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ErrorMsg(ErrorMsgEnum.ACCOUNT_NOT_FOUND.ErrorMsg));
+        RspAccount rspAccount = new RspAccount(account);
+        return ResponseEntity.status(HttpStatus.OK).body(rspAccount);
+    }
+
+    /**
+     *
+     * @param
+     * @return ResponseEntity
+     * @author jyf
+     */
     @Override
-    public Account getMyAccount(int id) {
-        return accountDao.getAccountById(id);
+    public ResponseEntity<Object> logoutService(String token){
```

Comment:
This `token` can be omitted here. It's not in use.
```suggestion
    public ResponseEntity<Object> logoutService(){
```